scripts/kubernetes/frontend_deployment.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: funcatron-frontend
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: funcatron-frontend
+    spec:
+      containers:
+        - name: frontend
+          resources:
+            requests:
+              cpu: 0.4
+              memory: 256Mi
+          #nodeSelector:
+          #  role: slave-public
+          imagePullPolicy: Always
```

Comment:
I set the imagePullPolicy to **Always** to match `"forcePullImage": true` in the Mesos config, but in general I prefer to use **IfNotPresent** to make pod startup faster, and rely on updating tags to get a newer version.